CODE_OF_CONDUCT.md

```diff
@@ -1,4 +1,11 @@
-## Code of Conduct
-This project has adopted the [Amazon Open Source Code of Conduct](https://aws.github.io/code-of-conduct).
-For more information see the [Code of Conduct FAQ](https://aws.github.io/code-of-conduct-faq) or contact
-opensource-codeofconduct@amazon.com with any additional questions or comments.
+# Code of Conduct
+
+We follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+<!-- TODO: Decide who will handle Code of Conduct reports and replace [INSERT EMAIL ADDRESS]
+    with an email address in the paragraph below. We recommend using a mailing list to handle reports.
+    If your project isn't prepared to handle reports, remove the project email address and just have
+    reporters send to conduct@cncf.io.
+-->
+Please contact [INSERT EMAIL ADDRESS] or the [CNCF Code of Conduct Committee](mailto:conduct@cncf.io)
```

Comment:
We can't leave this here because we aren't a CNCF project. I suggest removing the reporting mechanism until we have a mailing list, _or_ perhaps start with "Please message any project maintainer on Kubernetes slack"?